samples/instance.js

```diff
@@ -91,6 +95,15 @@ require('yargs')
   .example(
     'node $0 createInstanceWithProcessingUnits "my-instance" "my-project-id"'
   )
+  .command(
```

Comment:
why do we have update instance here, when we have created a new file ?

Reply:
updated the sample to rubric